crop-planner/src/api.ts

```diff
@@ -8,3 +8,12 @@ export const fetchFields = async (): Promise<Array<Field>> =>
 export const fetchCrops = async (): Promise<Array<Crop>> =>
   await fetch(`${SOIL_SERVICE_URL}/crops`).then(response => response.json())
 
+export const fetchHumusBalances = async (fields: Array<Field> = []): Promise<Array<HumusBalance>> =>
+  await fetch(`${SOIL_SERVICE_URL}/fields/humus_balance`, {
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    method: 'POST',
+    body: JSON.stringify({fields})
```

Comment:
Sending all fields data to BE here. Field ids and crop ids would suffice as all other data is already present on the BE for making humus balance calculations.